src/main/resources/jp/ikedam/jenkins/plugins/taglib/form/staticSelect/staticCombobox.js

```diff
@@ -50,12 +50,14 @@ Behaviour.register({"SELECT.staticCombobox": function(e) {
     orig.parentNode.removeChild(orig);
     
     /*
-     * Changes from the original 3:
-     *   Original behavior: Show candidates that matches the current incomplete input.
-     *   Changed behavior : Show all the candidates, not concerning with the current incomplete input.
+     * Changes from the original 4:
+     *   Original behavior: Show all the candidates, not concerning with the current incomplete input.
+     *   Changed behavior : Show candidates that matches the current incomplete input.

```

Comment:
The "original"means the combobox widget provided by Jenkins core.
https://github.com/jenkinsci/jenkins/blob/jenkins-2.89.3/core/src/main/resources/lib/form/combobox/combobox.js

The number of "Changes from the original" is an ordinal one, and you shouldn't change it.

Reply:
should i then revert the change number and update the commit?